### ISSUE TITLE
running with vitest not setting the `NODE_ENV` to `test`

### ISSUE THREAD

**Issue by rluvaton** [stryker-mutator/stryker-js#4289]
**Summary**

vitest set NODE_ENV to test, and when running mutation it does not

Same behavior as jest
#3580

**Stryker environment**

```
npm ls | grep stryker
├── @stryker-mutator/core@7.0.2
├── @stryker-mutator/vitest-runner@7.0.2
```

```
vitest@0.32.0
```

**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             | 16.18.1
| npm              | 8.19.2
| Operating System | macOs

**Comment by rluvaton** [comment]
creating a PR now

**Comment by nicojs** [comment]
Released in 7.1
